game_patch/object/obj_light.cpp

```diff
@@ -133,6 +134,25 @@ ConsoleCommand2 mesh_static_lighting_cmd{
     "Toggle mesh static lighting calculation",
 };
 
+CallHook<void(rf::Entity&)> entity_update_muzzle_flash_light_hook{
```

Comment:
I don't think it belongs to this file. `obj_light.cpp` is related to generating vertex colors for meshes so they become lit (think of it as a lightmap but less expensive).
This feature has nothing to do with that. The function name starts with `entity` so a good candidate would be `entity.cpp`. I generally try to layout the code the same as it was in original RF code. From IDA you can see that this function is close to other entity related functions. Keeping things grouped based on RF modules make it easier to avoid collisions between patches

Reply:
Good point, I will move it.

Reply:
You didn't move it

Reply:
I haven't made any changes to this PR yet.